content/departments/engineering/teams/source/strategy.md

```diff
@@ -1,3 +1,23 @@
-# Source Department Strategy
+# Source Strategy & Mission
 
-TL;DR: The Source department builds value for Sourcegraph by ensuring customers can index all of their code regardless of code host. We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
+## Strategy
+
+The Source team builds value for Sourcegraph by ensuring customers can index and secure all of their code regardless of code host and auth provider.
+
+We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
```

Comment:
```suggestion
The Source team builds value for Sourcegraph by ensuring customers can index and secure their code regardless of the code host and auth provider.

This is critical for building a long-term defensible value for Sourcegraph, as all other code-search solutions focus on searching across an individual repo or code host.
```

Just a suggestion to make this more straightforward to understand, In my opinion.